samples/manifests/plugins/dev-env-with-plugins.yaml

```diff
@@ -10,16 +10,16 @@ SharedEfsSgId: !SSM ${/orbit-f/dev-env/resources::SharedEfsSgId}
 InstallSsmAgent: true
 CognitoExternalProvider: !ENV ${COGNITO_EXTERNAL_PROVIDER::}
 CognitoExternalProviderLabel: !ENV ${COGNITO_EXTERNAL_PROVIDER_LABEL::}
-Images:
-    JupyterUser:
-        Repository: !ENV ${JUPYTER_USER_REPO::public.ecr.aws/v3o4w1g6/aws-orbit-workbench/jupyter-user}
-        Version: !ENV ${VERSION::latest}
-    OrbitController:
-        Repository: !ENV ${ORBIT_CONTROLLER_REPO::public.ecr.aws/v3o4w1g6/aws-orbit-workbench/orbit-controller}
-        Version: !ENV ${VERSION::latest}
-    UtilityData:
-        Repository: !ENV ${UTILITY_DATA_REPO::public.ecr.aws/v3o4w1g6/aws-orbit-workbench/utility-data}
-        Version: !ENV ${VERSION::latest}
+#Images:
```

Comment:
Can you remove the commented section

Reply:
I was leaving for reference...but can delete